colab-notebooks/README.md

```diff
@@ -0,0 +1,8 @@
+# Colabs
+  
+Links to interesting colabs demonstrating use of magenta for creative applications and also documentation for magenta. 
+
+[Mxml Documentation Colab](https://colab.research.google.com/github/tensorflow/magenta-demos/blob/master/colab-notebooks/MusicXML_Document_Structure_Documentation.ipynb) : 
```

Comment:
Let's standardize on MusicXML (rather than mxl, musicxml, etc.)

Reply:
Agreed, please use MusicXML, as that is the official name.

Reply:
Done https://github.com/tensorflow/magenta-demos/pull/64/commits/99cf09936934e0917199f2a76f509b78763642b0